infrastructure/keycloak/createuser.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+KEYCLOAK_REALM_URL="$KEYCLOAK_URL/auth/admin/realms/traffic-court"
+USERNAME="bobross"
+ADMIN_GROUP="traffic-court-admin"
+
+# Get admin user token
+ADMIN_TOKEN=$(curl -s --location --request POST "$KEYCLOAK_URL/auth/realms/master/protocol/openid-connect/token" \
+    --header 'Content-Type: application/x-www-form-urlencoded' \
+    --header 'Cookie: 8a49b93d4f41531bc1d1ef85f2b65254=2fb8f4c2fe6843d186204f21cc45dd4b' \
+    --data-urlencode 'client_id=admin-cli' \
+    --data-urlencode 'grant_type=password' \
+    --data-urlencode 'username=admin' \
+    --data-urlencode 'password=admin' | jq -r '.access_token')
+
+if [ -z "$ADMIN_TOKEN" ]
+then
+    echo "Error connecting to keycloak"
+    exit 1
+fi
+
+echo "Successfully connected to keycloak"
+
+# Create traffic-court-hub user
+CREATE_USER_STATUS=$(curl -s -o /dev/null -w '%{http_code}' \
+    --request POST "$KEYCLOAK_REALM_URL/users" \
+    --header "Authorization: Bearer $ADMIN_TOKEN" \
+    --header 'Content-Type: application/json' \
+    --data-raw '{
+                    "firstName": "bob",
+                    "lastName": "ross",
+                    "email": "bobross@paintit.com",
```

Comment:
`paintit.com` is a real domain. I usually prefer to use example.com or example.org as these are [RFC 2606](https://tools.ietf.org/html/rfc2606) reserved domain names that will never be used by real people.